app/components/Header/FancyNodesCanvas.js

```diff
@@ -63,15 +65,20 @@ class FancyNodesCanvas extends Component<Props, State> {
 
   drawGraphics() {
     const context = this._canvas.getContext('2d');
-    drawFancyNodes(context, {
-      width: this.state.width,
-      height: this.props.height,
+    if (this.state.animation) global.clearInterval(this.state.animation);
+
+    this.setState({
+      animation: drawFancyNodes(context, {
+        width: this.state.width,
+        height: this.props.height,
+      }),
```

Comment:
It was a bit confusing to me at first why you needed the animation in state, could you call it something else, perhaps `animationInterval`?.

Reply:
Good idea, i'll add it^^